src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/MetadataTests/DefaultJsonTypeInfoResolverTests.JsonTypeInfo.cs

```diff
@@ -72,7 +72,6 @@ public static void TypeInfoPropertiesDefaults(Type type)
             else
             {
                 Assert.Equal(0, ti.Properties.Count);
-                Assert.True(ti.Properties.IsReadOnly);
```

Comment:
why are you removing this? it seems correct

Reply:
Oops, accidental. Will revert.